nailgun/entities.py

```diff
@@ -2702,9 +2702,13 @@ def enable_rhrepo(self, base_arch,
             Return JSON response otherwise.
 
         """
+        data = {}
+        data[u'basearch'] = base_arch
```

Comment:
I don't think there is a need - base_arch is a must for all repos - so the calling code should make sure of that.  
With recent change in sat, release version is made optional and should not be passed with invalid or empty values.